### PR TITLE
feat: Added mantine link in frontend category

### PR DESCRIPTION
**Pull request by Avas77**
## Fixes Issue
Closes ##2221

## Changes proposed
Add the Mantine link in the frontend category.

## Screenshots
Screenshot 2023-12-25 at 20 37 39

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Hey @Avas77, is it only for react-components? if so, then please add this in `frontend/react.json` file

**Comment by Avas77** [comment]
Yes, It is a React component library that can be used with React-based frameworks Next.js, Gatsby, etc. Sure I will add it to react.json

**Review by rupali-codes — APPROVED** [review]
lgtm

File: database/frontend/react.json [hunk 1]
```diff
@@ -128,5 +128,12 @@
     "url": "https://www.coursera.org/learn/introduction-to-front-end-development/",
     "category": "frontend",
     "subcategory": "react"
+  },
+  {
+    "name": "Mantine",
+    "description": "Mantine is a React components library focused on providing great user and developer experience",
```
Comment (aftabrehan) [inline database/frontend/react.json:134]:
```suggestion
    "description": "Mantine is a React components library focused on providing great user and developer experience.",

```

**Review by aftabrehan — DISMISSED** [review]
Looks good to me! 👍 

Thanks for your contribution @Avas77.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
@rupali-codes @aftabrehan @CBID2 
I requested your reviews because it was dismissed due to new changes.

Review and merge :)

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**